### DISCUSSION TITLE
Migration guide: EncFS to GoCryptFS

### DISCUSSION THREAD

**Discussion by buhtz** [rfjakob/gocryptfs#795] [Q&A] [answered]
Is anyone aware of a migration guide for EncFS to GoCrypFS?
Or will anyone share experience about it?

I'm member of maintenance team of [Back In Time](https://github.com/bit-team/backintime) depending on EncFS. We are aware of the security problems and finguring out how to deal (https://github.com/bit-team/backintime/issues/1549 and https://github.com/bit-team/backintime/issues/1248) with it without breaking to much of our features.

**Comment by lechner** [comment]
Hi,

I'm not aware of a migration guide. Having migrated ten years ago or so myself, I also do not remember the steps. I believe it was a simple recursive copy ([possibly via tar](https://unix.stackexchange.com/a/43606)) from one mounted filesystem into the other.

The relationship with EncFS was always friendly, and the Gocryptfs author contributed there, as well. I would therefore be surprised if such a guide exists.

Kind regards
Felix

**Comment by rfjakob** [comment, marked as answer]
Even though I originally wrote this for migrating gocryptfs 0.x to gocryptfs 1.0, it basically applies to any filesystem migration, including EncFS to gocryptfs: https://github.com/rfjakob/gocryptfs/wiki/Upgrading
